envoyauth/response.go

```diff
@@ -114,6 +114,43 @@ func (result *EvalResult) IsAllowed() (bool, error) {
 	return false, result.invalidDecisionErr()
 }
 
+// GetRequestQueryParametersToRemove - returns the query parameters to remove from the original request before dispatching
+// it to the upstream
+func (result *EvalResult) GetRequestQueryParametersToRemove() ([]string, error) {
```

Comment:
Could you add a unit test for this method? Would be nice to make sure that changes don't regress in the future.

Reply:
Sure! I reworked `GetRequestHTTPHeadersToRemove` too bc `GetRequestQueryParametersToRemove` does the same.